pkg/pb/gracefulexit.proto

```diff
@@ -10,8 +10,44 @@ import "orders.proto";
 
 package gracefulexit;
 
+// NodeGracefulExit is a private service on storagenodes
+service NodeGracefulExit {
+  // GetSatellitesList returns a list of satellites that the storagenode has not begun a graceful exit for.
+  rpc GetNonExitingSatellites(GetNonExitingSatellitesRequest) returns (GetNonExitingSatellitesResponse);
```

Comment:
Why not return all the satellites with their current statuses? This would allow also showing the status from command line.

Reply:
That's a good idea. I will change that in my new PR